spec/unit/pops/types/p_object_type_spec.rb

```diff
@@ -490,21 +486,21 @@ def parse_object(name, body_string)
   end
 
   it 'raises an error when initialization hash contains invalid keys' do
-    expect_object('MyObject', <<-OBJECT)
+    obj = <<-OBJECT
       attribrutes => {
         a => Integer
       }
     OBJECT
-    expect { parser.parse('MyObject', loader) }.to raise_error(TypeAssertionError, /object initializer had wrong type, extraneous key 'attribrutes'/)
+    expect { parse_object('MyObject', obj) }.to raise_error(TypeAssertionError, /object initializer had wrong type, extraneous key 'attribrutes'/)
```

Comment:
Many users are not native english speakers, can we simplify "extraneous"? "unrecognized key", "invalid key" or something like that?